src/connectionTool/connections/IConnection.java

```diff
@@ -0,0 +1,13 @@
+package connectionTool.connections;
+
+import java.util.Properties;
+
+public interface IConnection {
+
+    String getConnectionString();
+    Properties getProperties();
+    String getHost();
+    int getTimeout();
+    String getPort();
```

Comment:
I removed this method from interface, because it wasn't used anywhere. I chose string, because port field is only used to create connection string, so I thought that casting it to int from config properties was unnecessary